src/microbe_stage/editor/upgrades/ChemoreceptorUpgradeGUI.cs

```diff
@@ -1,12 +1,24 @@
-﻿using System.Collections.Generic;
-using System.Linq;
+﻿using System;
+using System.Collections.Generic;
 using Godot;
 
 public class ChemoreceptorUpgradeGUI : VBoxContainer, IOrganelleUpgrader
 {
+    [Export]
+    public NodePath? TargetTypesPath;
```

Comment:
Only the first NodePath should be nullable. That's the style used in Thrive. So the other NodePaths here as well as the Dispose method need to be slightly modified to match Thrive style.